pkg/cloudproviders/aws/cloudprovider/launchtemplate.go

```diff
@@ -57,6 +58,7 @@ type LaunchTemplateProvider struct {
 	amiFamily             *amifamily.Resolver
 	securityGroupProvider *SecurityGroupProvider
 	cache                 *cache.Cache
+	logger                *zap.SugaredLogger
```

Comment:
Doesn't this lose automatic log level upgrading?

Reply:
Yep, I accidentally undid a previous PR, updated.